src/UnityExtension/Assets/Editor/GitHub.Unity/UI/HistoryView.cs

```diff
@@ -207,7 +207,8 @@ private void RefreshLog()
             {
                 if (Repository != null)
                 {
-                    GitClient.Log().ThenInUI((success, log) => {
+                    GitClient.Log().ThenInUI((success, log) =>
+                    {
```

Comment:
Sure, will fix that. I use tabs for spacing normally, so I had to reformat the code after I had done my changes, and thought that you guys just had missed this piece of formatting before. I will revert it. :)